src/mockSdk.js

```diff
@@ -34,6 +36,7 @@ export const mockSDKMeeting = {
         },
         person2: {
           id: 'mutedPerson',
+          name: 'SDK',
```

Comment:
let's use person names that contain firstnames and lastnames, to be closer to "reality". Maybe use the same names as here? https://github.com/webex/components/blob/master/src/data/people.js#L10